.github/workflows/CI.yml

```diff
@@ -843,11 +843,15 @@ jobs:
       - name: Package Windows Debug Info
         working-directory: build
```

Comment:
If it would be more convenient you could change the shell to `shell: msys2 {0}` and use bash... But I'm fine with this as is too.

Reply:
yeah, I was worried that might mess up the 7z.exe invocation, so I just stuck with PS.